web/ui/pagination-bar.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
```

Comment:
Can you create a story to check how it works?

Right now it is hard to tell how good `type="range"` fit the case (but I like the care of a11y here).